app/assets/stylesheets/frontend/styleguide/_colours.scss

```diff
@@ -19,34 +19,6 @@ $inside-gov-nav-dropdown: #888;
 $coalition-green: #85994b;
 $g8-brand: #c10019;
 
-// Web safe department colours
```

Comment:
Can you explain in this commit message please why these are removed, for future-us?